src/predict/jobs/predict_places_air_quality/main.py

```diff
@@ -33,11 +33,12 @@
     air_quality_readings_gdf = gpd.GeoDataFrame(air_quality_readings, geometry=geometry)
 
     airqlouds_gdf = get_shapefiles_gdf_from_gcs()
+    #sjoin
     merged_gdf = gpd.sjoin(
         airqlouds_gdf,
         air_quality_readings_gdf,
         how="left",
-        op="contains",
+        predicate="contains",
```

Comment:
Hi @wabinyai . thanks for this. 
Could you shed some more light on this? What exactly is going on here?